views/administration/forms/form-delete-alias.hbs

```diff
@@ -1,2 +1,2 @@
-<p>Sind Sie sich sicher, dass Sie folgendes Element löschen wollen?</p>
+<p>{{$t "administration.text.areYouSureYouWantToDeleteTheFollowingItem" }}</p>
```

Comment:
if we follow our naming schema. Doesn't it have to be 
```suggestion
<p>{{$t "administration.global.text.areYouSureYouWantToDeleteTheFollowingItem" }}</p>
```
because this text is visible on multiple administration pages?

This applies to most other administration keys here as well. Especially the keys inside administration/forms/*